src/main/java/org/nest/units/unitrepresentation/SIData.java

```diff
@@ -61,7 +61,7 @@
   private static String[] SIUnitsDerivedRaw =
       {"Hz","N","Pa","J","W","C","V","F","Ohm","S","Wb","T","H","lm","lx","Bq","Gy","Sv","kat"};
   private static String[] SIPrefixesRaw =
-      {"da","h","k","M","G","T","P","E","Z","Y","d","c","m","mu","n","p","f","a","z","y"};
+      {"k","M","G","T","P","E","Z","Y","m","mu","n","p","f","a","z","y"};
```

Comment:
This one is kind of silly: I dont have prefixes to express their higher powers. If you take hecto^2 you get a magnitude of 10^4 and I cannot convert that into an error message without a generic exponential notation. I have proposed such but until such time as there is a decision on it, these magnituded can lead to problems. 
Do you see any need for them specifically though?